### PR TITLE
Added draw of rectangle in 'onEmptyViewClicked'  and listener if the same date is clicked twice. (Similar to Google Calendar App).

### PR DESCRIPTION
**Pull request by asolis90**
Added functionality to draw rectangle in 'onEmptyViewClicked' and listener if the same date is clicked twice. Also added option for it to change the color of the rectangle. 
screenshot_2015-07-22-17-03-50

### CODE REVIEW

File: library/src/main/java/com/alamkanak/weekview/WeekView.java [hunk 1]
```diff
@@ -586,6 +613,57 @@ private void drawEvents(Calendar date, float startFromPixel, Canvas canvas) {
...
+     *
+     * @param mCacheEmptyEventDay           The day.
+     * @param canvas         The canvas to draw upon.
+     */
+    private void drawEmptyEvent(Calendar mCacheEmptyEventDay, Canvas canvas) {
+        // Prepare to iterate for each day.
+        Calendar day = (Calendar) mToday.clone();
+        day.add(Calendar.HOUR, 6);
+        int leftDaysWithGaps = (int) -(Math.ceil(mCurrentOrigin.x / (mWidthPerDay + mColumnGap)));
+        //iterate thru days
```
Comment (lolobosse) [inline library/src/main/java/com/alamkanak/weekview/WeekView.java]: Please correct english :smile: (feature is cool, though)

**Comment by lolobosse** [comment]
That's a cool feature but I do not think it has to be activated by default. I would love to have a boolean to activate or deactivate this mode directly in the XML (something like `activateDoubleClickListener`).

Cool work :tada: :+1: :exclamation:

**Comment by asolis90** [comment]
Thanks! :smile: I worked on it a couple days ago, wanted to share it with you guys. 
When you mean activate by boolean you mean the drawing of the rectangle and the doubleClickListener?

**Comment by lolobosse** [comment]
Yes that your code only works if I activate the feature (so you basically parse the `boolean` with a `false` as a default behaviour and then, you activate your `doubleClickListener` only if there this `boolean` is true).

**Comment by caske33** [comment]
Indeed, this should be an optional feature.

**Comment by wwahmed** [comment]
Any chance of this getting updated and merged -- it's a nice optional feature, but looks like it was developed on a much older revision and some updates might be necessary?

**Comment by asolis90** [comment]
Hey guys sorry about that been busy lately.. Will try to work on this feature this week..

**Comment by entropitor** [comment]
@asolis90 @jhoobergs is also working on this. He has a recent version.

**Comment by jhoobergs** [comment]
@asolis90 Here is my fork: https://github.com/Quivr/Android-Week-View/tree/Add_events
I think it is almost complete, but the text inside the rectangle isn't calculated properly

**Comment by asolis90** [comment]
hey guys, don't think i forgot about this feature.. lol Today I spent some time in it.. Its almost done and fully functional 😄

**Comment by entropitor** [comment]
@asolis90 @jhoobergs already has a working version. (One that's even in production already).

**Comment by wwahmed** [comment]
But is @jhoobergs planning to submit a pull request for it?

**Comment by asolis90** [comment]
I have tested his version and it is not the desired way to go about doing this. The empty event is conflicting with other events when its drawn, also the empty event rectangle should be drawn hourly and is not being handled correctly. I'll close this PR since its off track and open a new one. Please review my changes and if possible give some feedback.

**Comment by asolis90** [comment]
ref #321

**Pull request closed without merging**